### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by Vanethos**
Improvements in the ui

### CODE REVIEW

File: lib/presentation/utils/customized_animated_image.dart [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  _CustomizedAnimatedImageState createState() => _CustomizedAnimatedImageState();
+}
+
+class _CustomizedAnimatedImageState extends State<CustomizedAnimatedImage> with SingleTickerProviderStateMixin {
+
+  AnimationController _controller;
+
+  @override
+  Widget build(BuildContext context) {
+    var size = MediaQuery.of(context).size;
```
Comment (miguelpruivo) [inline lib/presentation/utils/customized_animated_image.dart]: This can `final`.

File: lib/presentation/ui/home/home_page.dart [hunk 2]
```diff
@@ -45,11 +45,11 @@ class _MyHomePageState extends BaseState<HomePage> {
     return Scaffold(
       key: scaffoldKey,
       body: StreamBuilder<int>(
-          initialData: widget.bloc.currentPageSubject.value,
-          stream: widget.bloc.currentPageStream,
-          builder: (context, snapshot) {
-            return _pages[snapshot.data];
-          }),
+            initialData: widget.bloc.currentPageSubject.value,
```
Comment (miguelpruivo) [inline lib/presentation/ui/home/home_page.dart]: Does this value can never be `null`? If so, it should be null aware protected.

File: lib/presentation/utils/customized_animated_image.dart (hunk 1 above)
Comment (miguelpruivo) [inline lib/presentation/utils/customized_animated_image.dart]: This can be `final` instead.

File: lib/presentation/ui/home/home_page.dart (hunk 2 above)
Comment (Vanethos) [reply lib/presentation/ui/home/home_page.dart]: Nope, never:
 `var currentPageSubject = BehaviorSubject<int>(seedValue: 0);`

**Comment by miguelpruivo** [comment]
@Vanethos feel free to resolve conflicts and merge as I don't own the permissions to do so on your branch.

**Comment by miguelpruivo** [comment]
@Vanethos there's still conflicts to solve.

**Comment by miguelpruivo** [comment]
@Vanethos there are still conflicts to solve.

**Comment by miguelpruivo** [comment]
@Vanethos there are still conflicts to address.

**Comment by Vanethos** [comment]
solved the conflicts

**Review by miguelpruivo — APPROVED** [review]
LGTM

**Pull request merged**